docs/make.jl

```diff
@@ -107,6 +107,7 @@ makedocs(
             ],
             "Time integration" => "time_integration.md",
             "Callbacks" => "callbacks.md",
+            "Coupling" => "coupling.md"
```

Comment:
```suggestion
            "Coupling" => "multi-physics_coupling.md"
```
In order to match the actual filename.